### PR TITLE
Add support for verbose(V) style logging

### PR DESCRIPTION
**Pull request by alexherrero**
Also a small typo fix.

### CODE REVIEW

**Review by pkern — APPROVED** [review]

File: logger.go [hunk 1]
```diff
@@ -281,6 +292,43 @@ func (l *Logger) Fatalf(format string, v ...interface{}) {
 	os.Exit(1)
 }
 
+// Verbose is a boolean type that implements Infof, etc.
+type Verbose bool
+
+// V generates a log record depends on the setting of the Level; or none default.
+func V(lvl Level) Verbose {
```
Comment (adjackura) [inline logger.go]: You also need to create these for Logger, you just created them for the default logger

**Review by adjackura — CHANGES_REQUESTED** [review]

File: logger.go (hunk 1 above)
Comment (alexherrero) [reply logger.go]: Done.

File: logger.go [hunk 2]
```diff
@@ -289,6 +338,22 @@ func SetFlags(flag int) {
 	defaultLogger.fatalLog.SetFlags(flag)
 }
 
+// SetLevel sets the verbosity level for verbose info logging in the
+// default logger.
+func SetLevel(lvl Level) {
+	defaultLogger.level = lvl
```
Comment (adjackura) [inline logger.go]: defaultLogger.SetLevel(lvl)

File: logger.go [hunk 3]
```diff
@@ -289,6 +338,22 @@ func SetFlags(flag int) {
...
+	defaultLogger.output(sInfo, 0, fmt.Sprintf("Info verbosity set to %d", lvl))
+}
+
+// V generates a log record depends on the setting of the Level or none
+// by default using the default logger.
+func V(lvl Level) Verbose {
+	return Verbose{
```
Comment (adjackura) [inline logger.go]: return defaultLogger.V(lvl)

**Review by adjackura — APPROVED** [review]

**Pull request merged**